src/test/java/org/betonquest/betonquest/utils/math/TokenizerTest.java

```diff
@@ -1,27 +1,30 @@
 package org.betonquest.betonquest.utils.math;
 
-import org.betonquest.betonquest.utils.logger.BetonQuestLogger;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.function.Executable;
-import org.mockito.MockedStatic;
 import org.betonquest.betonquest.BetonQuest;
 import org.betonquest.betonquest.api.Variable;
+import org.betonquest.betonquest.api.logger.util.BetonQuestLoggerValidationProvider;
 import org.betonquest.betonquest.config.Config;
 import org.betonquest.betonquest.config.ConfigPackage;
 import org.betonquest.betonquest.exceptions.InstructionParseException;
 import org.betonquest.betonquest.exceptions.QuestRuntimeException;
 import org.betonquest.betonquest.utils.math.tokens.Token;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.MockedStatic;
 
 import java.util.Collections;
 import java.util.Map;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.*;
```

Comment:
I see the point with disallowing star imports, but I believe that Assertions.* should be explicitly excluded from that rule, especially with `Mockito.*` still being a thing.